### ISSUE TITLE
Implement Stepper component

### ISSUE THREAD

**Issue by jalehman** [debois/elm-mdl#189]
I'm currently working on implementing the [Stepper component](https://material.google.com/components/steppers.html) using the css found in [this project](https://github.com/ahlechandre/mdl-stepper).

Since this project requires css that's not part of MDL, what is the recommended course of action for including it in elm-mdl (or not including it)?

By the way, great work on this project! Reading through the source has been immensely helpful in my efforts to learn elm.

Thank!

**Comment by debois** [comment]
I think the easier option is to just say "if you use the Stepper component, you must load this additional CSS", assuming the CSS in question is not in conflict with MDL CSS?

Other options are using `Options.stylesheet`, but I don't know if you can rely on that being loaded in time. 

One option that we'd rather avoid is using modified MDL CSS.

**Comment by jalehman** [comment]
Agreed -- instruction to load additional CSS seems to be the simplest approach. I have not found any conflicts between the two -- in fact the stepper CSS is written in the same style as MDL and this fits nicely.

**Comment by debois** [comment]
Great—looking forward to the PR!

**Comment by rerb** [comment]
@jalehman - how goes the stepper work?   I'd like to lend a hand, if I can.
